test/n_security_token_registry.js

```diff
@@ -1313,34 +1373,40 @@ contract("SecurityTokenRegistry", async (accounts) => {
 
         describe("Test cases for the setProtocolVersion", async () => {
             it("Should successfully change the protocolVersion -- failed because of bad owner", async () => {
-                await catchRevert(I_STRProxied.setProtocolFactory(accounts[8], 5, 6, 7, { from: account_temp }));
+                await catchRevert(I_STRProxied.setProtocolFactory(accounts[8], 5, 6, 7, { from: account_temp }),
+                    "Only owner");
             });
 
             it("Should successfully change the protocolVersion -- failed because factory address is 0x", async () => {
                 await catchRevert(
-                    I_STRProxied.setProtocolFactory(address_zero, 5, 6, 7, { from: account_polymath })
+                    I_STRProxied.setProtocolFactory(address_zero, 5, 6, 7, { from: account_polymath }),
+                    "Bad address"
                 );
             });
 
+            // @FIXME expected a revert regarding the version, but got "No factory".
```

Comment:
This test function is working correctly. That being said, we should update the description.
We say that this is not a valid version in the test case because there is no STFactory of that version. The version is actually a legal version but since there is no STFactory of that version attached, it can not be set as the latest version.